test/modules/core/index.js

```diff
@@ -18,12 +18,13 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import './utils';
-import './viewports';
-import './views';
-import './lifecycle';
+import './effects';
```

Comment:
FWIW - I see the appeal of an alphabetical list. Just to be clear, the previous order here was not random. The intention was to begin by unit testing more primitive classes before they were being used in more complicated classes, reflecting the "architectural layering" of the deck.gl core.

E.g. utils, views, viewports would be tested before lib, so the first errors in the log would be more likely to show where the issue actually was in a helper class, rather than a cascade of errors in the lib tests caused by a failure in the helper class. 

This probably doesn't matter much, but I do think that it is good to for deck devs to have a sense for this internal "layering", maybe it can be explained/emphasized in other ways.